src/modules/admin/components/Permissions/ColonyPermissionEditDialog.tsx

```diff
@@ -149,44 +176,52 @@ const ColonyPermissionEditDialog = ({
     [userData],
   );
 
-  const availableRoles = [
-    COLONY_ROLE_ROOT,
-    COLONY_ROLE_ADMINISTRATION,
-    COLONY_ROLE_ARCHITECTURE,
-    COLONY_ROLE_FUNDING,
-    COLONY_ROLE_ARBITRATION,
-  ].map(word => {
-    const roleKey = word.toLowerCase();
-    return capitalize(roleKey);
-  });
-
-  /* Currently arbitration should appear in the list of roles
-    but can not be set yet
-   */
-  const checkIfCanBeSet = role => {
-    if (role === 'Arbitration' || role === 'Root') {
-      return false;
-    }
-    return true;
-  };
+  // Get the current user's roles in the selected domain
```

Comment:
Same :-/

Reply:
Huh?

Reply:
Huh, that appeared on the wrong line... I was rather attempting to comment about the enum.